github/util.py

```diff
@@ -38,40 +38,41 @@ class GitHubRepositoryHelper(object):
 
     def __init__(
         self,
-        github: GitHub,
-        repository_owner: str,
-        repository_name: str,
+        github_cfg: GithubConfig,
+        owner: str,
+        name: str,
+        default_branch: str='master',
     ):
-        self.github = github
+        self.github = _create_github_api_object(github_cfg)
         self.repository = self._create_repository(
-            repository_owner=repository_owner,
-            repository_name=repository_name
+            owner=owner,
+            name=name
         )
+        self.default_branch = default_branch
 
-    def _create_repository(
-        self,
-        repository_owner: str,
-        repository_name: str
-    ):
+    def _create_repository(self, owner: str, name: str):
         repository = self.github.repository(
-                owner=repository_owner,
-                repository=repository_name
+                owner=owner,
+                repository=name
         )
         if not repository:
-            util.fail("Could not retrieve repository {owner}/{name}".format(owner=repository_owner, name=repository_name))
+            util.fail("Could not retrieve repository {owner}/{name}".format(owner=owner, name=name))
```

Comment:
do not use util.fail outside of cli/* . raise a RuntimeError instead (general rule)

in this case: github3.py raises an exception since version 1.1 - so rm the code block entirely